worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/PartitionDataWriter.java

```diff
@@ -166,24 +170,32 @@ public PartitionDataWriter(
   }
 
   public void initFileChannelsForDiskFile() throws IOException {
-    if (!this.diskFileInfo.isHdfs()) {
+    if (!this.diskFileInfo.isDFS()) {
       this.flusherBufferSize = localFlusherBufferSize;
       channel = FileChannelUtils.createWritableFileChannel(this.diskFileInfo.getFilePath());
     } else {
-      this.flusherBufferSize = hdfsFlusherBufferSize;
-      // We open the stream and close immediately because HDFS output stream will
+      FileSystem hadoopFs = null;
```

Comment:
Keep this hadoopFs as a class variable. No need to do so many conditions to get the corresponding hadoopFS.

Reply:
OK. Done